### PR TITLE
Task 2 - Creating Buttons and Inputs on design-system.html

### PR DESCRIPTION
**Pull request by JRRGomes**
**This PR resolves the Issue 2:**

**Task 2**

Now that the structure is in place, it's time to create some components we will be reusing to build the landing page. Create a **button with two variations, and an input with 4 variations**, all the states for hovered, pressed, active and disabled should be implemented as well.

[image](https://user-images.githubusercontent.com/11875867/152689426-10386146-30e7-4a3e-8088-49b0211b3f85.png)

Add examples for the components and all their variations on the **design-system.html page**.

### CODE REVIEW

**Comment by JRRGomes** [comment]
https://user-images.githubusercontent.com/19179208/153448537-6b366ea0-fbb9-49b8-85c6-ff6e43da39b2.mp4

File: src/design-system.html [hunk 1]
```diff
@@ -12,5 +12,19 @@
   </head>
   <body>
     <h1>Design System</h1>
+    <a href="/src/index.html">index</a>
+    <div>
+      <button class="button button--contact" >FIRST BUTTON</button>
```
Comment (romulostorel) [inline src/design-system.html]: 💅 
```suggestion
      <button class="button button--contact">FIRST BUTTON</button>
```

File: src/design-system.html [hunk 2]
```diff
@@ -12,5 +12,19 @@
...
+      <button class="button button--form" >SECOND BUTTON</button>
```
Comment (romulostorel) [inline src/design-system.html]: same here

File: exit [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+[36m@@ -1,3 +1,9 @@[m
+ :root {[m
+   --space-unit: 1em;[m
+[32m+[m[32m  --space-xx-small: calc(0.25*var(--space-unit));[m
+[32m+[m[32m  --space-x-small: calc(0.5*var(--space-unit));[m
+[32m+[m[32m  --space-small: calc(0.75*var(--space-unit));[m
+[32m+[m[32m  --space-medium: calc(1.25*var(--space-unit));[m
+[32m+[m[32m  --space-large: calc(2*var(--space-unit));[m
+[32m+[m[32m  --space-x-large: calc(3*var(--space-unit));[m
+ }[m
```
Comment (iagodahlem) [inline exit]: Acho que isso não devia estar aqui.

Comment (JRRGomes) [reply exit]: nao tenho idéia como criei isso hahaa acho que foi sem querer brigando com o terminal.

**Pull request merged**